docs/core/compatibility/sdk/8.0/version-requirements.md

```diff
@@ -9,6 +9,7 @@ Per the [published support rules](../../../porting/versioning-sdk-msbuild-vs.md#
 
 - 8.0.100 requires version 17.7 to be loaded but only supports targeting .NET 7 in that version.
 - 8.0.200 requires version 17.8.
+- 8.0.400 requires version 17.9.
```

Comment:
Hmm, it appears this got updated because nuget took a dependency on 17.9 and so to flow into the SDK, the minimum vs version got updated. For the purposes of this documentation, this is accurate.

However, we should review our policy and update this more intent as it doesn't match our other documentation here:
https://learn.microsoft.com/en-us/dotnet/core/porting/versioning-sdk-msbuild-vs#targeting-and-support-rules

@aortiz-msft do you recall why nuget bumped it's msbuild version in 6.11?

CC @baronfel @rainersigwald @jaredpar

Reply:
@zivkan , @nkolev92 ^

Reply:
> However, we should review our policy and update this more intent as it doesn't match our other documentation here

it would be great if the dotnet/sdk repo could also block insertions that violate the msbuild version, to minimize the risk of this kind of mistake happening again.

> do you recall why nuget bumped it's msbuild version in 6.11?

It looks like this was the PR that caused the version of MSBuild to change:
* https://github.com/NuGet/NuGet.Client/pull/5829

We couldn't write Apex tests for MAUI porjects without increasing the version of the VS Apex testing library, which is why that was upgraded. I'm guessing that MSBuild was a transitive dependency and we were getting package downgrade errors without also upgrading the MSBuild package versions.  Since NuGet.Client uses CPM, changing the MSBuild package version for our VS testing had the side effect of also changing our packages that the dotnet/sdk use.

Since we're servicing 8.0.4xx for another fix, I could try to downgrade MSBuild back to 17.9 if you like. Let me know. I'm pretty sure it means NuGet can't use a single MSBuild version for all projects in the repo, due to almost every single VS SDK component having a dependency on MSBuild

Reply:
@zivkan The SDK did [block ](https://github.com/dotnet/sdk/runs/25164865027) the insertion. Our vendor followed the instructions from the error message and updated and we didn't notice in our code review as we don't check codeflow PRs very closely.
https://github.com/dotnet/sdk/commit/abe622b40a7d54946028f667d2e7c0e91ec0d049
`Detected package downgrade: Microsoft.Build from 17.9.5 to centrally defined 17.8.3. Update the centrally managed package version to a higher version.`

I'm a bit confused by your reply as the current minimum version in 8.0.4xx is actually 17.9 but the PR you linked to updated nuget to 17.10. The minimum version per our policy should be 17.8. Not sure why we got downgrade errors when we updated to nuget version 6.11.0-preview.2.55 but were able to stay on 17.9.5 since then.
https://github.com/dotnet/sdk/blob/release/8.0.4xx/src/Layout/redist/minimumMSBuildVersion#L1

Reply:
Taking another look, a short time before Nikolche's PR, the MSBuild version was upgraded to fix another test issue:
* https://github.com/NuGet/NuGet.Client/pull/5805

Anyway, please let me know if you'd like me to try to downgrade MSBuild in NuGet's 6.11 branch and do another servicing fix into 8.0.4xx. I'm worried about how much effort it's going to take to not break tests, and I'm trying to focus on other deliverables for November's release, so I'll only attempt the MSBuild version downgrade if it's really needed.

Reply:
Since 8.0.4xx has only been out a short while, I don't know if we'd have gotten significant feedback either way. I think it's worth seeing if nuget can be downgraded to 17.8 and then the sdk can downgrade and match our documentation

Reply:
NuGet.Client's central package management is becoming less and less central due to conflicting requirements of our VS and .NET dependencies, but there's a fix incoming
* https://github.com/NuGet/NuGet.Client/pull/6015